man/systemd-nspawn.xml

```diff
@@ -307,7 +312,11 @@
         <option>--verity-data=</option> (and optionally <option>--root-hash-sig=</option>) options.</para>
 
         <para>Any other partitions, such as foreign partitions or swap partitions are not mounted. May not be specified
-        together with <option>--directory=</option>, <option>--template=</option>.</para></listitem>
+        together with <option>--directory=</option>, <option>--template=</option>.</para>
+
+        <para>In case the name of the parent directory ends with <filename>auto.d</filename> suffix, the path
+        is searched for items starting with the path but using versioned suffixes. The highest version is
```

Comment:
highest → newest

Reply:
OK, though "newest" could be interpreted as "newest mtime", but that's not the case, especially for devices.

Reply:
fixed as "newest"